migrations/V132__Add_TRB_Request_LCIDs_Table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE trb_request_lcids (
+    id UUID PRIMARY KEY NOT NULL,
+    trb_request_id uuid NOT NULL REFERENCES trb_request(id),
+    lcid TEXT NOT NULL CHECK (lcid ~ '^[A-Z]?[0-9]{6}$'),
```

Comment:
I wonder if it makes sense to FKEY to the intake itself, now that we're returning that

For all intents and purposes, the FE would just display LCID and request name, but if we want to show the mapping we see here, I think we'll need to maintain a connection from the TRB request to the intake request, effectively

![Screenshot 2023-03-24 at 10 34 19 AM](https://user-images.githubusercontent.com/15203744/227554782-efb6d57e-debe-4743-a655-0401bb0d5292.png)

Reply:
We'd probably just change the mutation to take an intake request ID too, and have associated intakes with TRB requests instead of just the LCID?

Not sure how much that complicates things, but I think this UI showing the request name and the LCID means we'll have to tie to the request as a whole, not just the LCID